tests/pytests/unit/states/test_nftables.py

```diff
@@ -0,0 +1,528 @@
+                )
+
+                mock = MagicMock(
+                    side_effect=[
+                        {"result": False, "comment": ""},
+                        {"result": True, "comment": ""},
+                        {"result": True, "comment": ""},
+                    ]
+                )
+                with patch.dict(nftables.__salt__, {"nftables.check_chain": mock}):
+                    ret.update(
+                        {
+                            "comment": "Failed to flush chain  in table"
+                            "  in family ipv4, chain does not exist."
+                        }
+                    )
+                    assert (
+                        nftables.flush("salt", table="", chain="", ignore_absence=False)
+                        == ret
+                    )
+
+                    mock = MagicMock(
+                        side_effect=[
+                            {"result": True, "comment": ""},
+                            {"result": False, "comment": ""},
+                        ]
+                    )
+                    with patch.dict(nftables.__salt__, {"nftables.flush": mock}):
+                        ret.update(
+                            {
+                                "changes": {"locale": "salt"},
+                                "comment": "Flush nftables rules in  table  chain ipv4 family",
+                                "result": True,
+                            }
+                        )
+                        assert nftables.flush("salt", table="", chain="") == ret
+
+                        ret.update(
+                            {
+                                "changes": {},
+                                "comment": "Failed to flush nftables rules",
+                                "result": False,
+                            }
+                        )
+                        assert nftables.flush("salt", table="", chain="") == ret
+
+
+def test_set_policy():
+    """
+    Test to sets the default policy for nftables firewall tables
+    """
+    ret = {"name": "salt", "changes": {}, "result": True, "comment": ""}
+
+    mock = MagicMock(return_value=[])
+    with patch.object(nftables, "_STATE_INTERNAL_KEYWORDS", mock):
+        mock = MagicMock(return_value="stack")
+        with patch.dict(nftables.__salt__, {"nftables.get_policy": mock}):
+            ret.update(
+                {
+                    "comment": "nftables default policy for chain"
+                    "  on table  for ipv4 already set to stack"
+                }
+            )
+            assert (
+                nftables.set_policy("salt", table="", chain="", policy="stack") == ret
+            )
+
+            with patch.dict(nftables.__opts__, {"test": True}):
+                ret.update(
+                    {
+                        "comment": "nftables default policy for chain"
+                        "  on table  for ipv4 needs to be set to sal",
```

Comment:
```suggestion
                        "comment": "nftables default policy for chain  on table  for ipv4 needs to be set to sal",
```

Reply:
line is too long for this case, black complains

Reply:
line is too long for black

Reply:
then
```
"comment": (
    "nftables default policy for chain  on table  for "
    "ipv4 needs to be set to sal"
)
```
it's more readable

Reply:
Fixed